src/pages/workspaces/workspace/Analyses.js

```diff
@@ -260,23 +258,29 @@ const Analyses = _.flow(
     return getFileName(name)
   }, analyses)
 
-  //TODO: defined load function for azure
-  const refreshAnalyses = !!googleProject ? _.flow(
-    withRequesterPaysHandler(onRequesterPaysError),
-    withErrorReporting('Error loading analyses'),
-    Utils.withBusyState(setBusy)
-  )(async () => {
+  const loadGoogleAnalyses = async () => {
     const rawAnalyses = await Ajax(signal).Buckets.listAnalyses(googleProject, bucketName)
     const notebooks = _.filter(({ name }) => _.endsWith(`.${tools.Jupyter.ext}`, name), rawAnalyses)
     const rAnalyses = _.filter(({ name }) => _.includes(getExtension(name), tools.RStudio.ext), rawAnalyses)
 
     //we map the `toolLabel` and `updated` fields to their corresponding header label, which simplifies the table sorting code
-    const enhancedNotebooks = _.map(notebook => _.merge(notebook, { application: tools.Jupyter.label, lastModified: notebook.updated }), notebooks)
-    const enhancedRmd = _.map(rAnalysis => _.merge(rAnalysis, { application: tools.RStudio.label, lastModified: rAnalysis.updated }), rAnalyses)
+    const enhancedNotebooks = _.map(notebook => _.merge(notebook, { application: tools.Jupyter.label, lastModified: new Date(notebook.updated).getTime() }), notebooks)
+    const enhancedRmd = _.map(rAnalysis => _.merge(rAnalysis, { application: tools.RStudio.label, lastModified: new Date(rAnalysis.updated).getTime() }), rAnalyses)
```

Comment:
Why is this step not needed in `loadAzureAnalyses`? Are these fields already on Azure blobs?

Reply:
i moved this logic to the service layer for azure